### PR TITLE
Editprofile, Application id and workspace settings

### PR DESCRIPTION
**Pull request by Lord-Chris**
Workspace settings

https://user-images.githubusercontent.com/58702861/137355004-4ac1eddf-8e32-4a1a-b0df-01df73a129e7.mp4

### CODE REVIEW

File: lib/package/base/server-request/api/zuri_api.dart [hunk 1]
```diff
@@ -808,19 +816,18 @@ class ZuriApi implements Api {
...
+      "file": MultipartFile(
         image!.openRead(),
         await image.length(),
         filename: image.path.split(Platform.pathSeparator).last,
       ),
     });
     try {
       final res = await dio.post(
-        'https://api.zuri.chat/organizations/$orgId/members/$memberId/photo',
+        'https://api.zuri.chat/upload/file/$pluginId',
```
Comment (pauleke65) [inline lib/package/base/server-request/api/zuri_api.dart:830]: use $coreBaseUrl here

File: lib/ui/view/organization/organization_settings/organization_icon.dart [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+class OrganizationLogo extends StatelessWidget {
+  final OrganizationModel org;
+  const OrganizationLogo({Key? key, required this.org}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return ViewModelBuilder<OrganizationSettingsViewModel>.reactive(
+      viewModelBuilder: () => OrganizationSettingsViewModel(),
+      builder: (context, model, child) {
+        return Scaffold(
```
Comment (pauleke65) [inline lib/ui/view/organization/organization_settings/organization_icon.dart:21]: Please add a screenshot for this particular widget.
I don't get why an appBar is here

File: lib/ui/view/organization/organization_settings/organization_name_url.dart [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+import 'package:hng/constants/app_strings.dart';
+import 'package:hng/models/organization_model.dart';
+import 'package:hng/ui/shared/colors.dart';
+import 'package:hng/ui/shared/styles.dart';
+import 'package:hng/ui/shared/text_field.dart';
+import 'package:hng/ui/shared/zuri_appbar.dart';
+import 'package:hng/ui/shared/zuri_loader.dart';
+import 'package:hng/ui/view/organization/organization_settings/organization_settings_view_model.dart';
+
+class OrganizationNameUrl extends HookWidget {
```
Comment (pauleke65) [inline lib/ui/view/organization/organization_settings/organization_name_url.dart:13]: This is meant to extend to HookViewModelWidget not HookWidget

Comment (pauleke65) [reply lib/ui/view/organization/organization_settings/organization_name_url.dart:13]: HookViewModelWidget<OrganizationSettingsViewModel>

File: lib/ui/view/organization/organization_view/organization_view.dart [hunk 4]
```diff
@@ -147,21 +148,25 @@ class OrganizationTile extends ViewModelWidget<OrganizationViewModel> {
             height: MediaQuery.of(context).size.height * 0.05,
             width: MediaQuery.of(context).size.height * 0.05,
             decoration: BoxDecoration(
-              border: Border.all(
-                color: AppColors.whiteColor,
-                width: 2,
-                style: BorderStyle.solid,
-              ),
-              color: AppColors.greyishColor,
+              // border: Border.all(
```
Comment (pauleke65) [inline lib/ui/view/organization/organization_view/organization_view.dart:151]: Commented codes to be removed

File: lib/ui/view/organization/organization_settings/organization_name_url.dart (hunk 3 above)
Comment (Lord-Chris) [reply lib/ui/view/organization/organization_settings/organization_name_url.dart]: Done

File: lib/package/base/server-request/api/zuri_api.dart (hunk 1 above)
Comment (Lord-Chris) [reply lib/package/base/server-request/api/zuri_api.dart]: Done

**Review by pauleke65 — APPROVED** [review]
LG

**Review by jeremiahseun — APPROVED** [review]
LGTM

**Pull request merged**